### PR TITLE
Add a new SDK NVIDIA Nsight Aftermath

### PR DESCRIPTION
**Pull request by iRebbok**
Games using NVIDIA Nsight Aftermath:
- [Squad](https://steamdb.info/app/393380)
- [The Immortal Mayor](https://steamdb.info/app/1426730)
- [Tom Clancy's Rainbow Six® Siege](https://steamdb.info/app/359550)
- [Splitgate](https://steamdb.info/app/677620)

I added a rule for the SDK in alphabet order that matches the library for Linux & Windows including the x64 & x86 arches.

### CODE REVIEW

**Comment by xPaw** [comment]
Here's the actual names in use, so the regex can be simplified.
```
    [gfsdk_aftermath_lib.x64.dll] => 5043
    [gfsdk_aftermath_lib.dll] => 1102
    [gfsdk_aftermath_lib.x86.dll] => 6
    [gfsdk_aftermath_lib_uwp.x64.dll] => 1
```

File: descriptions/SDK.NVIDIA_Nsight_Aftermath.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[**NVIDIA® Nsight™ Aftermath**](https://developer.nvidia.com/nsight-aftermath) is a library for DirectX 12 or Vulkan game crash reporters that generates GPU "mini-dumps" when a Time-Out Detection and Recovery or exception occurs.
```
Comment (xPaw) [inline descriptions/SDK.NVIDIA_Nsight_Aftermath.md:1]: Remove ®/™

**Comment by iRebbok** [comment]
Does it mean there are files without an arch?

**Comment by xPaw** [comment]
Yes.

File: descriptions/SDK.NVIDIA_Nsight_Aftermath.md (hunk 1 above)
Comment (iRebbok) [reply descriptions/SDK.NVIDIA_Nsight_Aftermath.md]: Done.

File: rules.ini [hunk 2]
```diff
@@ -113,6 +113,7 @@ FMOD = (?:^|/)(?:lib)?fmod(?:l|ex|exl|studio|studiol|)(?:64)?\.(?:dylib|dll|so)$
 NodeJS = (?:^|/)node\.dll$
 NVIDIA_Ansel = (?:^|/)AnselSDK(?:32|64)\.dll$
 NVIDIA_DLSS = (?:^|/)nvngx_dlss\.dll$
+NVIDIA_Nsight_Aftermath = (?:^|/)?GFSDK_Aftermath_Lib(?:\.x86|\.x64)?\.dll$
```
Comment (xPaw) [inline rules.ini:116]: Remove the question mark after `(?:^|/)`

**Pull request merged**